partiql-lang/src/main/kotlin/org/partiql/lang/eval/builtins/BuiltinImplems.kt

```diff
@@ -0,0 +1,9 @@
+package org.partiql.lang.eval.builtins
+
+import org.partiql.lang.eval.ExprValue
+import org.partiql.lang.eval.stringValue
+
+internal fun characterLength(value: ExprValue): Int {
+    val str = value.stringValue()
+    return str.codePointCount(0, str.length)
+}
```

Comment:
Rather than making a new file, perhaps we put this in ExprValue extensions as "stringValueCodepointLength"

```kotlin
private fun ExprValue.stringValuecodepointLength(): Int {
    val str = stringValue()
    return str.codePointCount(0, str.length)
}
```

Alternatively, since all builtin classes are internal, this can be a static function of `ExprFunctionCharLength`. I missed that on the initial factoring.

Any particular reason why we are adding ExprFunctionLength? Seems confusing because the input domain isn't included in the function name. There are also already two functions that do this. We also have two functions for the lengths of collections (cardinality (sql) and size (?) ). I don't think we need this one.